bin/conjur-cli.rb

```diff
@@ -131,8 +131,8 @@ def test_select
   DESC
   cgrp.command :generate do |c|
     c.action do |global_options,options,args|
-      exit_now! "No command arguments are allowed" unless args.empty?
```

Comment:
When you remove "No command arguments are allowed", when you run `conjurctl data-key generate lol wat` the output is
```
error: parse error: 
```

If, instead of your update, you just replace `exit_now` with `help_now` (and leave everything else the same), then it outputs
```
error: parse error: No command arguments are allowed
```
before the help info.

I think this is clearer. There are several commands impacted by similar updates.

Reply:
I second that. BTW - since exiting because no args were expected looks like a common pattern - you might want to consider creating a utility method or a dedicated exception class.